src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets check and update the License headers in all the places.

Reply:
Fixed